blockmanager.go

```diff
@@ -2173,6 +2180,13 @@ func (b *blockManager) handleNotifyMsg(notification *blockchain.Notification) {
 		}
 
 		for _, tx := range block.STransactions()[0:] {
+			// Remove rebroadcast inventory for revocations
+			if txType := stake.DetermineTxType(tx.MsgTx()); txType == stake.TxTypeSSRtx {
```

Comment:
Same here regarding this being incorrect.  A block being disconnected does not necessarily invalidate a revocation.